### PR TITLE
support external Compiled Predicate call

### PR DESCRIPTION
**Pull request by syuhei176**

### CODE REVIEW

File: packages/ovm/src/decompiler/CompiledPredicate.ts [hunk 1]
```diff
@@ -206,6 +206,22 @@ export const createAtomicPropositionCall = (
   } else if (input.predicate.type == 'VariablePredicateCall') {
     // When predicateDef has VariablePredicate, inputs[1] must be variable name
     return FreeVariable.from(def.inputs[1] as string)
+  } else if (input.predicate.type == 'CompiledPredicateCall') {
```
Comment (syuhei176) [inline packages/ovm/src/decompiler/CompiledPredicate.ts:209]: type "ExternalCall" is more suitable?

Comment (tkmct) [reply packages/ovm/src/decompiler/CompiledPredicate.ts:209]: do you mean "ExternalCall" instead of "CompiledPredicateCall"?

Comment (tkmct) [reply packages/ovm/src/decompiler/CompiledPredicate.ts:209]: Is it actually `External`? I'm not sure about this point.

Comment (syuhei176) [reply packages/ovm/src/decompiler/CompiledPredicate.ts:209]: If we borrow the word from Solidity, it's external contract call which creates EVM call.

Comment (tkmct) [reply packages/ovm/src/decompiler/CompiledPredicate.ts:209]: It makes sense. thank you for explaining

**Pull request merged**